spec/features/page_redirects_spec.rb

```diff
@@ -255,12 +252,12 @@
 
       it "if requested page is unpublished" do
         visit "/not-public"
-        expect(page.current_path).to eq("/public-child")
+        expect(page.current_path).to eq("/not-public/public-child")
```

Comment:
This looks a little wrong. The page has a telling URL of `not-public`, but still shows up in the URL? Do I smell more code that can be deleted here? `redirect_to_public_child?`?

Reply:
Yes, we might consider that. I have the same feeling about that feature, but I need to recap why we introduced that feature in the first hand. SEO is hard and weird in many ways.

Definitely something to tackle in a separate PR.